apps/prairielearn/src/pages/instructorInstanceAdminAccess/instructorInstanceAdminAccess.html.ts

```diff
@@ -38,7 +38,9 @@ export function InstructorInstanceAdminAccess({
 
           <div class="card mb-4">
             <div class="card-header bg-primary text-white d-flex align-items-center">
-              ${course_instance.long_name} course instance access rules
+              <h1 class="h6 font-weight-normal mb-0">
+                ${course_instance.long_name} course instance access rules
```

Comment:
This isn't a problem with this PR, but we're kinda inconsistent in how these headings are constructed. I see three variants:

- "LONG NAME course instance access rules" (this page)
- "LTI configuration" and similar
- "Course instance SHORT NAME" (the settings page)

Not requesting a change here, but it'd be nice to be more consistent for folks who are using screen readers. Maybe another issue to track this?

Reply:
I agree. Opened issue #10447 to track this.